src/test/java/net/openhft/chronicle/threads/EventGroupTest.java

```diff
@@ -327,6 +327,10 @@ public void testCloseAddHandler() {
         }
     }
 
+    private static void close(EventLoop eventGroup) {
```

Comment:
why this change? Seems unnecessary

Reply:
Added a comment to make it clearer why something like this is needed